benchmark/sdpa_benchmark_training/configs/qwen35.py

```
@@ -36,7 +36,7 @@
     # fa4 rows are kept (they fail) to document the sm100 kernel limitation.
     data_types=["bfloat16"],
     attn_masks=["top_left"],  # Causal only
-    profile_pass="fwd",  # Forward-only (bwd blocked at head_dim=256)
+    profile_pass="both",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "profile_pass"

(S106)